docs/user-guide/getting-into-containers.md

```diff
@@ -0,0 +1,49 @@
+﻿#Getting into containers: kubectl exec
+Developers can use `kubectl exec` to run commands in a container. This guide demonstrates two use cases.
+
+##Using kubectl exec to check the environment variables of a container
+Kubernetes exposes [services](../../docs/services.md#environment-variables) through environment variables. It is convenient to check these environment variables using `kubectl exec`.
+
+
+We first create a pod and a service,
+```
+$ kubectl create -f examples/guestbook/redis-master-controller.yaml
+$ kubectl create -f examples/guestbook/redis-master-service.yaml
+```
+wait until the pod is Running and Ready,
+```
+$ kubectl get pod
+NAME                 READY     REASON       RESTARTS   AGE
+redis-master-ft9ex   1/1       Running      0          12s
+```
+then we can check the environment variables of the pod, 
+```
+$ kubectl exec redis-master-ft9ex env

```

Comment:
probably should add a shell command as well because that might be the most common use case and requires -ti option
```
$ kubectl exec -ti redis-master-ft9ex -- /bin/bash
```